SaveVideo\2023_02_17\2023_02_17.mp4
-i "SaveVideo\2023_02_17\2023_02_17.mp4" -ss -24562 -t 10 "SaveVideo\2023_02_17\checkin_E28069950000400BF371357D_2023_02_17.mp4"
1
=>17/02/2023 3:17:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

SaveVideo\2023_02_17\2023_02_17.mp4
-i "SaveVideo\2023_02_17\2023_02_17.mp4" -ss -24562 -t 10 "SaveVideo\2023_02_17\checkin_E28069950000400BF371357D_2023_02_17.mp4"
1
=>17/02/2023 3:17:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

SaveVideo\2023_02_17\2023_02_17.mp4
-i "SaveVideo\2023_02_17\2023_02_17.mp4" -ss -24562 -t 10 "SaveVideo\2023_02_17\checkin_E28069950000400BF371357D_2023_02_17.mp4"
1
=>17/02/2023 3:17:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

SaveVideo\2023_02_17\2023_02_17.mp4
-i "SaveVideo\2023_02_17\2023_02_17.mp4" -ss -24562 -t 10 "SaveVideo\2023_02_17\checkin_E28069950000400BF371357D_2023_02_17.mp4"
1
=>17/02/2023 3:17:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

SaveVideo\2023_02_17\2023_02_17.mp4
-i "SaveVideo\2023_02_17\2023_02_17.mp4" -ss -24562 -t 10 "SaveVideo\2023_02_17\checkin_E28069950000400BF371357D_2023_02_17.mp4"
1
2
=>17/02/2023 3:17:01 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

2
2
2
3
3
3
3
4
4
4
4
5
5
5
5
6
6
6
6
7
7
7
7
8
8
8
8
9
9
9
9
10
10
10
10
11
11
11
11
12
12
12
12
13
13
13
13
14
14
14
14
15
15
15
15
16
16
16
16
17
17
17
18
18
18
19
19
19
20
20
20
21
21
21
22
22
22
23
23
23
24
24
24
25
25
25
26
26
26
27
27
27
28
28
28
29
29
29
30
30
30
oke
oke
oke
=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:00 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

=>17/02/2023 3:22:01 pm An Error occurred:    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.API_odoo.<APIGetTimeCheckoutcutVideoAsync>d__4.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\API_odoo.cs:line 60
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at RFIDAttendance.MainForm.<dailyCutVideo>d__19.MoveNext() in C:\RFID\rfid\rfid-develop_webpos_rfid\rfid-develop_attendance_rfid\RFIDAttendance\MainForm.cs:line 180  Message: Response status code does not indicate success: 500 (INTERNAL SERVER ERROR).

false
false
false
false
false
false
false
false
false
08:36:34
08:36:35
false
12:05:21
false
false
false
false
false
false
false
false
false
12:26:49
14:10:36
SaveVideo\2023_02_17\2023_02_17.mp4
-i "SaveVideo\2023_02_17\2023_02_17.mp4" -ss 631 -t 10 "SaveVideo\2023_02_17\checkout_E28069950000500AC144D4E5_2023_02_17.mp4"
14:09:57
SaveVideo\2023_02_17\2023_02_17.mp4
-i "SaveVideo\2023_02_17\2023_02_17.mp4" -ss 592 -t 10 "SaveVideo\2023_02_17\checkout_E28069950000400AC144D4FF_2023_02_17.mp4"
1
14:10:13
SaveVideo\2023_02_17\2023_02_17.mp4
-i "SaveVideo\2023_02_17\2023_02_17.mp4" -ss 608 -t 10 "SaveVideo\2023_02_17\checkout_E28069950000500AC144D51D_2023_02_17.mp4"
1
1
false
false
false
false
false
false
14:10:50
SaveVideo\2023_02_17\2023_02_17.mp4
-i "SaveVideo\2023_02_17\2023_02_17.mp4" -ss 645 -t 10 "SaveVideo\2023_02_17\checkout_E28069950000400AC144D501_2023_02_17.mp4"
1
2
2
false
false
14:08:16
SaveVideo\2023_02_17\2023_02_17.mp4
-i "SaveVideo\2023_02_17\2023_02_17.mp4" -ss 491 -t 10 "SaveVideo\2023_02_17\checkout_E28069950000500BF371358A_2023_02_17.mp4"
1
2
3
3
2
3
4
4
3
4
false
false
